composer.json

```diff
@@ -7,7 +7,7 @@
     "require": {
         "php": ">=7.2.0",
         "ext-json": "*",
-        "oat-sa/lib-lti1p3-core": "^4.0",
+        "oat-sa/lib-lti1p3-core": "dev-feature/service-handlers",
```

Comment:
not ready for merge

Reply:
now based on core 5.0.0, we can merge once approved